_pages/web-ta.md

```diff
@@ -0,0 +1,129 @@
+
+People with disabilities navigate the web in a variety of ways.  People who are blind may use screen readers, which are devices that speak the text that appears on a screen.  People who are deaf or hard of hearing may use captioning.  And people whose disabilities affect their ability to grasp and use a mouse may use voice recognition software to control their computers and other devices with verbal commands.
+
+The ways that websites are designed and set up can create unnecessary barriers that make it difficult or impossible for people with disabilities to use websites, just as physical barriers like steps can prevent some people with disabilities from entering a building.  These barriers on the web keep people with disabilities from accessing information and programs that businesses and state and local governments make available to the public online.  But these barriers can be prevented or removed so that websites are accessible to people with disabilities.
+
+## Examples of Website Accessibility Barriers
+
+- **Poor color contrast.**  People with limited vision or color blindness cannot read text if there is not enough contrast between the text and background (for example, light gray text on a light-colored background).  
+
+- **Use of color alone to give information.**  People who are color-blind may not have access to information when that information is conveyed using only color cues because they cannot distinguish certain colors from others.  Also, screen readers do not tell the user the color of text on a screen, so a person who is blind would not be able to know that color is meant to convey certain information (for example, using red text alone to show which fields are required on a form).  
+
+- **Lack of text alternatives (“alt text”) on images.**  People who are blind will not be able to understand the content and purpose of images, such as pictures, illustrations, and charts, when no text alternative is provided.  Text alternatives convey the purpose of an image, including pictures, illustrations, charts, etc.  
+
+- **No captions on videos.**  People with hearing disabilities may not be able to understand information communicated in a video if the video does not have captions.
+
+- **Inaccessible online forms.**  People with disabilities may not be able to fill out, understand, and accurately submit forms without things like:
+    - Labels that screen readers can convey to their users (such as text that reads “credit card number” where that number should be entered);
```

Comment:
@c2nelson The Markdown linter is reporting that lines 32, 33 & 34 are indented more than expected. Change the indentation from 4 spaces to 2 spaces and re-run.